packages/core/src/connectors/coinbaseWallet.ts

```diff
@@ -37,8 +37,7 @@ export class CoinbaseWalletConnector extends Connector<
 > {
   readonly id = 'coinbaseWallet'
   readonly name = 'Coinbase Wallet'
-  readonly ready =
-    typeof window !== 'undefined' && !window.ethereum?.isCoinbaseWallet
+  readonly ready = true
```

Comment:
`CoinbaseWalletConnector` was coupled to `InjectedConnector` so that only one would be active at a time. `CoinbaseWalletConnector` should always work either via QR code modal and/or the Coinbase Wallet browser extension so it can always be set to `true`.